src/Compilers/VisualBasic/Portable/Symbols/BaseTypeAnalysis.vb

```diff
@@ -190,7 +190,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
             Debug.Assert(root.OriginalDefinition = root, "root must not be a substitution")
 
-            If current Is Nothing Then
+            If current Is Nothing Or TypeOf current Is MissingMetadataTypeSymbol Then
```

Comment:
@AlekseyTs The error location is [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Symbols/Source/SourceMemberContainerTypeSymbol.vb,2226).  When building in debug mode, I hit the first assert.